spot_driver/launch/spot_driver.launch.py

```diff
@@ -336,45 +139,21 @@ def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
 
     ld.add_action(rviz)
 
-    # Parse config options to create a list of composable node descriptions for the nodelets we want to run within the
-    # composable node container.
-    composable_node_descriptions = (
-        create_depth_registration_nodelets(context, spot_name, has_arm)
-        if depth_registered_mode is DepthRegisteredMode.FROM_NODELETS
-        else []
-    ) + (create_point_cloud_nodelets(context, spot_name, has_arm) if publish_point_clouds else [])
-    container = launch_ros.actions.ComposableNodeContainer(
-        name="container",
-        namespace=spot_name,
-        package="rclcpp_components",
-        executable="component_container_mt",
-        output="screen",
-        composable_node_descriptions=composable_node_descriptions,
+    spot_image_publishers = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource([FindPackageShare(THIS_PACKAGE), "/launch", "/spot_image_publishers.launch.py"]),
+        launch_arguments={
+            "config_file": LaunchConfiguration("config_file"),
+            "depth_registered_mode": LaunchConfiguration("depth_registered_mode"),
+            "publish_point_clouds": LaunchConfiguration("publish_point_clouds"),
+            "uncompress_images": LaunchConfiguration("uncompress_images"),
+            "publish_compressed_images": LaunchConfiguration("publish_compressed_images"),
+            "stitch_front_images": LaunchConfiguration("stitch_front_images"),
+            "spot_name": LaunchConfiguration("spot_name"),
+        }.items(),
```

Comment:
```suggestion
        launch_arguments={
            key: LaunchConfiguration(key) for key in ["config_file", "depth_registered_mode", "publish_point_clouds", "uncompress_images", "publish_compressed_images", "stitch_front_images", "spot_name"]
        }.items(),
```

Reduces a bit of boilerplate when a new one needs to be added